src/test/scala/nl/knaw/dans/easy/schemaExamples/SchemaValidationFixture.scala

```diff
@@ -15,19 +15,128 @@
  */
 package nl.knaw.dans.easy.schemaExamples
 
-import java.net.UnknownHostException
+import java.net.{ URL, UnknownHostException }
 
-import better.files.StringOps
+import better.files.{ File, StringOps }
 import javax.xml.XMLConstants
 import javax.xml.transform.Source
 import javax.xml.transform.stream.StreamSource
 import javax.xml.validation.{ Schema, SchemaFactory }
+import org.scalatest.prop.{ TableDrivenPropertyChecks, TableFor1 }
 import org.scalatest.{ FlatSpec, Matchers }
 
-import scala.util.{ Failure, Try }
-import scala.xml.{ Elem, SAXParseException }
+import scala.util.{ Failure, Success, Try }
+import scala.xml._
 
-trait SchemaValidationFixture extends FlatSpec with Matchers {
+trait SchemaValidationFixture extends FlatSpec with Matchers with TableDrivenPropertyChecks {
 
+  val publicSchema: String
+  val localSchemaFile: String
+  val examples: TableFor1[String]
 
+  private val publicEasySchemaBase = "//easy.dans.knaw.nl/schemas"
+  protected val httpsEasySchemaBase = s"https:$publicEasySchemaBase"
+
+  private lazy val triedPublicSchema: Try[Schema] = Try {
+    SchemaFactory
+      .newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+      .newSchema(Array(new StreamSource(publicSchema.toString)).toArray[Source])
+  }
+  private lazy val triedLocalSchema: Try[Schema] = Try {
+    // lazy for two reasons:
+    // - schemaFile is set by concrete test class
+    // - postpone loading until actually validating
+    val xsdInputStream = File(localSchemaFile).contentAsString.replaceAll(
+      s"""schemaLocation="http:$publicEasySchemaBase""",
+      s"""schemaLocation="file://$schemaDir"""
+    ).inputStream
+    SchemaFactory
+      .newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+      .newSchema(Array(new StreamSource(xsdInputStream)).toArray[Source])
+  }
+
+  "examples" should "be schema valid with local copy of easy-schema" in {
+    forEvery(examples) { example =>
+      val xml = XML.loadFile((exampleDir / example).toString())
+      easyLocationsIn(xml).foreach(File(_).toJava should exist)
```

Comment:
1. if you add `implicit def existenceOfFile[FILE <: better.files.File]: Existence[FILE] = _.exists` to the top of this file (first thing inside the trait), you don't need the `.toJava` anymore
2. would it be nice to have `easyLocationsIn` return `better.files.File`?
3. if you apply the above changes, you can write this line like `all(easyLocationsIn(xml)) should exist`, which would probably give you a better error message if something fails (not fail-fast). Side-note, there is an `all` and an `every`; I'm not sure yet what the difference between them is... feel free to experiment a bit 😉

Reply:
The all and every popped up before.  All reported only the first broken test, every reports every.

Reply:
ah, now I remember! Then you should use `every`, right?

Reply:
My answer applied to line 59, you are talking about 61....

Reply:
@1 implicit is overkill, it affect the subclasses to.

Reply:
I don't get what you mean. If you add `implicit def existenceOfFile[FILE <: better.files.File]: Existence[FILE] = _.exists` to the package object and replace `easyLocationsIn(xml).foreach(File(_).toJava should exist)` with `every(easyLocationsIn(xml)) should exist`, you get a way better result, right? Could you clarify what you mean with 

> it affects the subclasses to.